lib/plugins/insulinage.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+var _ = require('lodash');
+var moment = require('moment');
+var times = require('../times');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['times' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1335229634/issues/source?bid=2452379&fileBranchId=2957759#l5)